spec/features/signed_in_visitor_creates_a_recipe_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/features/clearance_helpers'
+
+RSpec.feature 'A signed in user creates a recipe' do
+  scenario 'with valid parameters' do
+    user = create(:user)
+    recipe = FactoryBot.build(:recipe)
+
+    sign_in_with user.email, user.password
+    visit '/'
+    click_link 'New Recipe'
+    fill_in 'Title', with: recipe.title
+    fill_in 'Ingredients', with: recipe.ingredients
+    fill_in 'Instructions', with: recipe.instructions
+    click_button 'Create'
+
+    expect(page).to have_current_path('/')
+  end
+end
```

Comment:
In modern versions of Rails (5+ IIRC)  I think it's preferred to create these kinds of specs under `spec/system` rather than `spec/features`. This is because "system tests" became first-class feature in Rails whereas before the concept of "feature tests" was just tacked on by the community.